tests/suites/test_suite_pkparse.data

```diff
@@ -1198,7 +1198,7 @@ pk_parse_key:"30070201010400a000":MBEDTLS_ERR_PK_KEY_INVALID_FORMAT
 
 Key ASN1 (OneAsymmetricKey X25519, doesn't match masking requirements, from RFC8410 Appendix A but made into version 0)
 depends_on:MBEDTLS_PK_HAVE_ECC_KEYS:PSA_WANT_ECC_MONTGOMERY_255
-pk_parse_key:"302e020100300506032b656e04220420f8ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3f":0
+pk_parse_fix_x25519:"302e020100300506032b656e04220420f8ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3f":"302e020100300506032b656e04220420f8ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7f"
```

Comment:
Thanks for adding this, but when it comes to tests, I seem to always want more :)

1. Pre-existing: this tests that we are setting the most significant bit (last byte (= most significant, this is little-endian) changed from 3f to 7f), but doesn't check that we are unsetting the least significant bits (first / least significant byte was already f8). (Note: we can keep this test as is, as it's from the RFC, but add another one where the first byte is ff initially.) (Actually, after reading the source again, our testing of the most significant byte is not complete either: for the 2nd test, we should try with `af` as the most significant byte, should also be changed to `7f`.)
2. As you guessed, I'd like to test x448 too :) We can generate a valid key with `openssl genpkey -algorithm x448 -outform der | xxd -p` (replace `| xxd -p` with your favourite way of converting bin to hex if it's not installed on your machine). The interesting part is the last 56 bytes. I think we want to replace them with `7f ff .... ff ff` which should be changed to `ff ff ... ff f4` if I'm counting correctly. (And if we test x448 too, we should rename this to `pk_parse_fix_montgomery`.

Reply:
As for point (1) TBH this is actually I tried yesterday (I mean, setting the bits that were supposed to be fixed with the wrong polarity) but then the parsing was failing so I gave up. I suspect that I might not done it properly, so I will give it a second try.

Instead, for point (2), thanks for the suggestions. I'll do it ;)